presto-hive/src/main/java/com/facebook/presto/hive/orc/OrcPageSource.java

```diff
@@ -213,11 +219,13 @@ protected void closeWithSuppression(Throwable throwable)
         private final int columnIndex;
         private final Type type;
         private boolean loaded;
+        private final FileFormatDataSourceStats stats;
```

Comment:
place this after `type` so the field ordering and constructor args are consistent